src/components/elements/youtube-video-pill.tsx

```diff
@@ -81,7 +81,7 @@ const YoutubeVideoPillBounded = ({videoUrl, ...props}: Props) => {
     >
       <YouTube
         videoId={videoId || shortId}
-        className={"mx-auto w-fit overflow-hidden transition-all duration-300 ease-in-out " + (isPlaying && !isInitialPlay ? "" : "rounded-full")}
+        className={"*:w-full *:aspect-[1/2] *:h-full overflow-hidden transition-all duration-300 ease-in-out " + (isPlaying && !isInitialPlay ? "" : "rounded-full outline outline-4 outline-white outline-offset-[-10px]")}
```

Comment:
There is `iframeClassName` you could use instead of `*:`

Reply:
I don't really like that aspect ratio. It leaves a black bar at the top and bottom 
![Uploading image.png…]()